code/modules/maps/spawners/spawners.dm

```diff
@@ -1000,3 +1000,142 @@
 		/obj/item/weapon/storage/firstaid/regular,
 		/obj/item/weapon/storage/box/handcuffs,
 		)
+
+
+// Safe /////////////////////////////////////////////////////
+//Does not come with a safe.
+/obj/map/spawner/safe/any
+	name = "safe any spawner "
+	icon_state = "safe"
+	to_spawn = list(
```

Comment:
This isn't a criticism of your code here because this is just the way this specific code works but we should definitely be cutting back on the amount of lists we have and replacing them with procs, especially things at round start for sweet performance increases